### PR TITLE
ZL/ZR to change IP by 10 at a time

### PR DESCRIPTION
**Pull request by PKPenguin321**
If the IP you need to change is, say, 120, you'll have to push the Up key on your Switch 119 times to reach it. With this PR, you'll only have to push ZR 12 times and the Down key once. That's a difference of over a hundred button presses, which is really nice.

Side note, I did ~~not~~ test this ~~but the code seems simple enough that it should work~~

### CODE REVIEW

File: Switch_Client/source/main.c [hunk 1]
```diff
@@ -126,6 +127,15 @@ int main(int argc, char* argv[]) {
...
 		if (kDown & KEY_DDOWN && currentIpBlock == 3) ipBlock4--;
 
+	    	if (kDown & KEY_ZR && currentIpBlock == 0) ipBlock1+=10;
+		if (kDown & KEY_ZL && currentIpBlock == 0) ipBlock1-=10;
+		if (kDown & KEY_ZR && currentIpBlock == 1) ipBlock2+=10;
+		if (kDown & KEY_ZL && currentIpBlock == 1) ipBlock2-=10;
+		if (kDown & KEY_ZR && currentIpBlock == 2) ipBlock3+=10;
+		if (kDown & KEY_ZL && currentIpBlock == 2) ipBlock3-=10;
+		if (kDown & KEY_ZR && currentIpBlock == 3) ipBlock4+=10;
+		if (kDown & KEY_ZL && currentIpBlock == 3) ipBlock4-=10;
```
Comment (PKPenguin321) [inline Switch_Client/source/main.c]: another sidenote, this monster of an if() chain could definitely be cut down a bit, but i'm kind of lazy... and also it only needs to run until the switch connects to the IP in question, so it should be fine

**Comment by WerWolv** [comment]
This could be done by having two switch cases testing for the current block wrapped in one if for ZR and ZL. Would be a bit prettier imo.

**Comment by PKPenguin321** [comment]
I'm looking at just making the four ipBlock variables into one array right now

**Comment by PKPenguin321** [comment]
I also tested it this time so I can confirm that this works

**Comment by WerWolv** [comment]
Looking a lot nicer now. Thanks:)

**Pull request merged**